app/controllers/api/subcollections/configuration_script_payloads.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module ConfigurationScriptPayloads
+      def configuration_script_payloads_query_resource(object)
+        object.respond_to?(:configuration_script_payloads) ? object.configuration_script_payloads : []
```

Comment:
Do not all ConfigurationScripts respond to configuration_script_payloads? If they don't, can you add a test showing that this check is necessary?